compiler/test/dotty/tools/vulpix/ParallelTesting.scala

```diff
@@ -825,7 +825,11 @@ trait ParallelTesting extends RunnerOrchestration { self =>
       def sawDiagnostic(d: Diagnostic): Unit =
         d.pos.nonInlined match
           case srcpos if srcpos.exists =>
-            val key = s"${relativize(srcpos.source.file.toString)}:${srcpos.line + 1}"
+            // If an annotated // error is placed on the last line of the file, EOF errors can show up past the last.
+            // Without this adjustment, it's not possible to annotate // error on errors whose position is EOF
+            // in some cases.
```

Comment:
I think I add anypos-error for positions that can't be expressed in comment.

https://github.com/lampepfl/dotty/commit/dff31fb0766c237d43fea07b413c302f23d8a9b3

Reply:
You did, and I'll happily use that if it's better. I think it's a little unfortunate not to be able to add a `// error` at the end of a file to indicate that the error should be at the end of the file though? I agree this isn't  great, but I'd rather not use `anypos` all else equal. Happy to if this is what gets the PR through!

Reply:
(I reverted this change because I realized how to get the existing reporting mechanism to be happy).